src/main/java/ch/njol/skript/expressions/ExprCommandInfo.java

```diff
@@ -0,0 +1,146 @@
+	"command \"sk\"'s plugin owner"})
+@Since("INSERT VERSION")
+public class ExprCommandInfo extends SimpleExpression<String> {
+	
+	static {
+		Skript.registerExpression(ExprCommandInfo.class, String.class, ExpressionType.SIMPLE,
+			"[the] main [command] [name] of command %string%", "command %string%'s main [command] [name]",
+			"[the] description of command %string%", "command %string%'s description",
+			"[the] label of command %string%", "command %string%'s label",
+			"[the] usage of command %string%", "command %string%'s usage",
+			"[the] aliases of command %string%", "command %string%'s aliases",
+			"[the] permission of command %string%", "command %string%'s permission",
+			"[the] permission message of command %string%", "command %string%'s permission message",
+			"[the] plugin [owner] of command %string%", "command %string%'s plugin [owner]");
+	}
+	
+	@SuppressWarnings("null")
+	InfoType type;
+	@SuppressWarnings("null")
+	Expression<String> commandName;
+
+	@Override
+	public boolean init(Expression<?>[] exprs, int matchedPattern, Kleenean isDelayed, ParseResult parseResult) {
+		commandName = (Expression<String>) exprs[0];
+		type = InfoType.values()[Math.floorDiv(matchedPattern, 2)];
+		return true;
+	}
+	
+	@Nullable
+	@Override
+	@SuppressWarnings("null")
+	protected String[] get(Event e) {
+		String commandName = this.commandName.getSingle(e);
+		if (commandName == null)
+			return null;
+		commandName = commandName.toLowerCase().split(" ")[0];
+		if (commandName.startsWith("/"))
+			commandName = commandName.substring(1);
+		Command command = Commands.getCommandMap().getCommand(commandName);
+		if (command == null)
+			return null;
+		switch (type) {
+			case NAME:
+				return new String[]{command.getName()};
+			case DESCRIPTION:
+				return new String[]{command.getDescription()};
+			case LABEL:
+				return new String[]{command.getLabel()};
+			case USAGE:
+				return new String[]{command.getUsage()};
+			case ALIASES:
+				return command.getAliases().toArray(new String[0]);
+			case PERMISSION:
+				return new String[]{command.getPermission()};
+			case PERMISSION_MESSAGE:
+				return new String[]{command.getPermissionMessage()};
+			case PLUGIN:
+				if (command instanceof PluginCommand) {
+					return new String[]{((PluginCommand) command).getPlugin().getName()};
+				} else if (command instanceof BukkitCommand) {
+					return new String[]{"Bukkit"};
+				} else if (command.getClass().getPackage().getName().startsWith("org.spigot")) {
+					return new String[]{"Spigot"};
+				} else if (command.getClass().getPackage().getName().startsWith("com.destroystokyo.paper")) {
+					return new String[]{"Paper"};
+				}
+		}
+		return new String[0];
+	}
+	
+	@Override
+	public boolean isSingle() {
+		return true;
+	}
+	
+	@Override
+	public Class<? extends String> getReturnType() {
+		return String.class;
+	}
+	
+	@Override
+	public String toString(@Nullable Event e, boolean debug) {
+		return commandName + " command";
```

Comment:
```suggestion
		return "the " + type.name().toLowerCase().replace("_", "") + " of command " + commandName.toString(e, debug)
```

*may require some slight changes to the enum to make this work*